From b770d54ababc96d756082912bd6b2b277e366892 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:26 -0500
Subject: [redhat] kabi: Adding symbol pci_iounmap (lib/iomap.c)

Message-id: <380bb9843539a357c8cce0289899faa8d4660974.1418903403.git.poros@redhat.com>
Patchwork-id: 102263
O-Subject: [PATCH RHEL7.1 34/48] kabi: Adding symbol pci_iounmap (lib/iomap.c)
Bugzilla: 1160762
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160762
    https://bugzilla.redhat.com/show_bug.cgi?id=1160762

    Description:
    ============
    Used in cleanup of unable to asign nevice a minor number
    used to tear down bar0 mmap when removing a particular instance of a PCIE-RT card.
    Used in pcie_rt.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/pci_iounmap
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/pci_iounmap
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/pci_iounmap
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pci_iounmap

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/pci_iounmap b/redhat/kabi/kabi-module/kabi_ppc64/pci_iounmap
new file mode 100644
index 0000000..d48dba3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/pci_iounmap
@@ -0,0 +1,2 @@
+#1-
+0xc2171b99 pci_iounmap vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/pci_iounmap b/redhat/kabi/kabi-module/kabi_ppc64le/pci_iounmap
new file mode 100644
index 0000000..de2965f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/pci_iounmap
@@ -0,0 +1,2 @@
+#1-
+0xb246dd27 pci_iounmap vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/pci_iounmap b/redhat/kabi/kabi-module/kabi_s390x/pci_iounmap
new file mode 100644
index 0000000..eafb686
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/pci_iounmap
@@ -0,0 +1,2 @@
+#1-
+0x7596ca5c pci_iounmap vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pci_iounmap b/redhat/kabi/kabi-module/kabi_x86_64/pci_iounmap
new file mode 100644
index 0000000..e6eecbc
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pci_iounmap
@@ -0,0 +1,2 @@
+#1-
+0x127b8725 pci_iounmap vmlinux EXPORT_SYMBOL
-- 
1.7.1